src/ring_theory/class_group.lean

```diff
@@ -62,6 +62,19 @@ variables {R K}
   (to_principal_ideal R K x : fractional_ideal R⁰ K) = span_singleton _ x :=
 rfl
 
+lemma to_principal_ideal_mul (x y : Kˣ) :
+  to_principal_ideal R K x * to_principal_ideal R K y = to_principal_ideal R K (x * y) :=
+by simpa only [ext_iff, coe_to_principal_ideal] using span_singleton_mul_span_singleton _ _
```

Comment:
This is just `map_mul`? `to_principal_ideal` is a monoid_hom, so this lemma isn't needed.